packages/commonwealth/client/scripts/views/pages/discussions/HeaderWithFilters/HeaderWithFilters.tsx

```diff
@@ -32,6 +32,9 @@ type HeaderWithFiltersProps = {
   totalThreadCount: number;
   isIncludingSpamThreads: boolean;
   onIncludeSpamThreads: (includeSpams: boolean) => any;
+  isIncludingArchivedThreads: boolean;
+  onIncludeArchivedThreads: (includeArchived: boolean) => any;
```

Comment:
`onIncludeArchivedThreads` looks like a bool toggle, can we do `onArchiveChange: (includeArchived: boolean) => any`

Reply:
I feel like that obfuscates the action, which is that the user is trying to include archived threads alongside non-archived threads.